src/components/filter/ExperienceFilter.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { Button } from "@/components/ui/button";
+import { Calendar22 } from "@/components/ui/calendarFilter";
+
+const experienceTypes = ["Quartos", "Eventos", "Laboratórios", "Trilhas"];
```

Comment:
No backend o enum de categorias de experiência tem os seguintes valores:

TRAIL
HOSTING
LABORATORY
EVENT

Tu chegou a conversar com o resto do time de frontend para verificar como o pessoal planeja utilizar esse componente? Porque se for manter esses nomes (`["Quartos", "Eventos", "Laboratórios", "Trilhas"]`) será preciso mapear o original no backend.

Reply:
Ou mapear no próprio componente

Reply:
Não precisa ser com alguém do backend, só procura quem for utilizar o componente e verifica onde vai ficar o mapeamento desses caras, se tiverem alguma dúvida vejam com o @jhbergallo ou o @kaitowing.